openshift/overrides/hack/openshift-pipelines.sh

```diff
@@ -62,13 +62,6 @@ wait_pipelines_ready() {
   fi
 }
 
-tekton_tasks() {
-  echo "Creating Pipeline tasks..."
-  oc apply -f ${source_path}/pkg/pipelines/resources/tekton/task/func-deploy/0.1/func-deploy.yaml
-  oc apply -f ${source_path}/pkg/pipelines/resources/tekton/task/func-s2i/0.1/func-s2i.yaml
-  oc apply -f ${source_path}/pkg/pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks.yaml
-}
-
 tasks_only=false
```

Comment:
you can also remove these lines below that checks for installing tasks only and just keep the sentences
```
openshift_pipelines
wait_pipelines_ready

echo Done
```

Reply:
https://github.com/openshift-knative/kn-plugin-func/pull/792/files/d239a16cbab0ebf89affa33103828c04db94273a#diff-3aa06581077dbf2b051f12deea17060e51a3c3e76734f0de181b962593bb618eL72-R75

Reply:
resolved